public/app/features/plugins/admin/components/VersionInstallButton.tsx

```diff
@@ -150,6 +151,16 @@ function getIcon(label: string) {
   return '';
 }
 
+function isDowngrade(installedVersion?: string, version?: string): boolean | undefined {
+  if (!installedVersion || !version) {
+    return undefined;
+  }
+  if (!valid(installedVersion) || !valid(version)) {
+    return undefined;
+  }
+  return gt(installedVersion, version);
+}
+
```

Comment:
I get a bit confused by the 3-state of the return value. Maybe we should return an enum instead, that might make things more clear. WDYT?